apps/base-docs/docs/notices/preparing-for-fault-proofs-on-base.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Preparing for fault proofs on Base Sepolia
-slug: /preparing-for-fault-proofs-on-base-sepolia
-description: Fault proofs are expected to go live for Base Sepolia testnet in mid-July.
+title: Preparing for fault proofs on Base
+slug: /preparing-for-fault-proofs-on-base
+description: Fault proofs are expected to go live on Base Mainnet in late October / early November.
```

Comment:
let's update all instances now to just `late October`

Reply:
c3ebf304